File: String_Functions/Frequency_Map/freq.py
def freqMap(arr):
    dc = {}
    for i in arr:
        dc[i] = dc.get(i,0) + 1
    return dc

if __name__ == "__main__":
    arr = [1, 2, 3, 1]
    mp = freqMap(arr)
    for k, v in mp.items():
        print(k,v)

# For cf contests, use string as keys, to prevent your code from hackers

File: String_Functions/Frequency_Map/freq.cpp
#include <iostream>
#include <vector>
#include <unordered_map>

using namespace std;

template <typename T>
unordered_map<T, int> freqMap(vector<T> arr){
    unordered_map<T,int> dc;
    for (T i: arr)
        dc[i]++;
    return dc;
}


int main()
{
    vector<int> vc = {1,2,3};
    unordered_map<int,int> mp = freqMap(vc);
    for (auto [p,q]: mp)
        cout<<p<<" "<<q<<'\n'; 
    return 0;
}

// For cf contests, use ordered map, to prevent your code from hackers